src/Bladeburner/utils/loadActionIdentifier.ts

```diff
@@ -23,3 +23,49 @@ export function loadActionIdentifier(identifier: unknown): ActionIdentifier | nu
   }
   return null;
 }
+
+/** These shorthands match those documented in the BB Terminal Help */
+export const BladeBurnerTerminalShorthands = {
+  contracts: <readonly string[]>["contract", "contracts", "contr"],
+  operations: <readonly string[]>["operation", "operations", "op", "ops"],
+  blackops: <readonly string[]>[
+    "blackoperation",
+    "black operation",
+    "black operations",
+    "black op",
+    "black ops",
+    "blackop",
+    "blackops",
+  ],
+  general: <readonly string[]>["general", "general action", "gen"],
+} as const;
+
+export function fuzzyActionIdentifier(typeShorthand: string, name: string): ActionIdentifier | null {
```

Comment:
BladeBurnerTerminalShorthands and resolveBladeBurnerAction do not need to be exported.
In your tests you can use `(Bladeburner as any).privateMethodOrField`.

Reply:
Oh, I see, you moved those methods into a separate file. In that case, using `export` is fine. However, I don't agree with the filename "load...". It would be better to use something like "resolve..." or "match..." instead.

Reply:
Yeah I moved it to a file that had cohesive logic and kept it there till I learn more. These type lookups occur on a few more places so am looking for a simpler naming for the entire thing.